mackup/application.py

```diff
@@ -124,7 +146,12 @@ def restore(self):
                                               home_filepath))
                 and utils.can_file_be_synced_on_current_platform(filename)):
 
-                print("Restoring {} ...".format(home_filepath))
+                print("Restoring {} ...".format("" if self.verbose else filename))
```

Comment:
again here, a simple "if verbose: ... else: ..." would make things easier to read.